grapher/dataTable/DataTable.tsx

```diff
@@ -212,15 +213,26 @@ export class DataTable extends React.Component<{
             const actualColumn = dim.coreTableColumn
             const unit =
                 actualColumn.unit === "%" ? "percent" : dim.coreTableColumn.unit
+            const targetTime =
+                dim.columns.length == 1 ? dim.columns[0].targetTime : undefined
             const columnName =
                 actualColumn.displayName !== ""
                     ? actualColumn.displayName
                     : actualColumn.name
 
             const dimensionHeaderText = (
                 <React.Fragment>
-                    <span className="name">{upperFirst(columnName)}</span>
-                    <span className="unit">{unit}</span>
+                    <div className="name">{upperFirst(columnName)}</div>
+                    <div>
+                        <span className="unit">{unit}</span>{" "}
+                        <span className="divider">
+                            {unit && targetTime !== undefined && "•"}
```

Comment:
nit: might be nice to use the proposition (e.g. `in` or `on`) instead of a bullet? Only if it's easy 😬

Reply:
I thought about that! But I don't think it always fits in super nicely with the unit, and I like the visual distinction between the two.

One example where it's not great is "Total tests": there the unit is `Tests performed`, and if we make it `Tests performed on May 29, 2022` it becomes misleading.

Reply:
Good point! Makes sense to keep it separated with a •